src/index.html

```diff
@@ -15,8 +15,29 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="/"class="logo__img"><img src="./images/logo.png"alt="Moyo Logo"/></a>
```

Comment:
There should not be spaces within the `href` attribute value of the `<a>` tag. Also, ensure that the `class` attribute is separated from the `href` attribute by a space.